sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -798,6 +800,18 @@ public void shutdown() {
         suspend(false, new CancellationException());
     }
 
+    @Override
+    public void sendLogs() {
+        for (GroupState groupState : mGroupStates.values()) {
+            checkPendingLogs(groupState);
+        }
+    }
+
+    @Override
+    public void suspendSendLogs(boolean deleteLogs) {
```

Comment:
No need to expose `deleteLogs` parameter, we have `shutdown` function for `false`. Remove parameter and rename